node/flows.re

```diff
@@ -294,6 +294,32 @@ let received_signature = (state, update_state, ~hash, ~signature) => {
 let received_operation =
     (state, update_state, request: Networking.Operation_gossip.request) =>
   if (!List.mem(request.operation, state.Node.pending_side_ops)) {
+    let.ok () =
+      Operation.Side_chain.(
+        switch (request.operation.kind, request.signature) {
+        | (Add_validator(_), Some(signature))
+        | (Remove_validator(_), Some(signature)) =>
+          let.assert () = (
+            `Invalid_signature_author,
+            Address.compare(
+              state.Node.identity.t,
+              Signature.public_key(signature),
+            )
+            == 0,
+          );
+          let payload_hash =
+            request.operation
+            |> Operation.Side_chain.to_yojson
+            |> Yojson.Safe.to_string
+            |> BLAKE2B.hash;
+          if (payload_hash |> Signature.verify(~signature)) {
+            Ok();
+          } else {
+            Error(`Failed_to_verify_payload);
+          };
+        | _ => Ok()
+        }
+      );
```

Comment:
This change should definitely be in another PR.

Reply:
Opened https://github.com/marigold-dev/deku/pull/228 to address this